openshift/topics/advanced_concepts.adoc

```diff
@@ -405,6 +405,68 @@ When restarting the server it is important to restart just the JBoss EAP process
 ====
 . Log in to the `master` realm's administration console of the {project_name} server using the credentials created in the steps above. In the browser, navigate to *\http://sso-<project-name>.<hostname>/auth/admin*  for the {project_name} web server, or to *\https://secure-sso-<project-name>.<hostname>/auth/admin* for the encrypted {project_name} web server, and specify the user name and password used to create the administrator user.
 
+
+=== Customizing the default behavior of the {project_name} image
+
+You can change the default behavior of the {project_name} image such as enabling technology preview features or enabling debugging. This section describes how to make this change by using the JAVA_OPTS_APPEND variable.
+
+.Prerequisites 
+
+This procedure assumes that the {project_openshift_product_name} image has been previously link:{project_doc_base_url}/red_hat_single_sign-on_for_openshift/index#Example-Deploying-SSO[deployed using one of the following templates:]
```

Comment:
Please use full link instead of `project_doc_base_url` in this case similarly the case of other sections. This link is causing failures in tests.

```suggestion
This procedure assumes that the {project_openshift_product_name} image has been previously link:https://access.redhat.com/documentation/en-us/red_hat_single_sign-on_continuous_delivery/7/html-single/red_hat_single_sign-on_for_openshift/index#Example-Deploying-SSO[deployed using one of the following templates:]
```

Reply:
Thanks, @hmlnarik That link would need to be conditional on the document being in the CD version of documents.   However, we may get a ruling in the next few days to discontinue CD releases because almost no one uses them.  If that happens, I recommend we use the full path for 7.3:

https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.3/html-single/red_hat_single_sign-on_for_openshift/index#Example-Deploying-SSO